dkms.in

```diff
@@ -875,10 +875,10 @@ prepare_signing()
                     sign_file="/usr/src/linux-headers-$kernelver/scripts/sign-file"
                 fi
                 ;;
-            * )
-                sign_file="/lib/modules/$kernelver/build/scripts/sign-file"
-                ;;
         esac
+        if [[ ! -f "${sign_file}" ]]; then
+            sign_file="/lib/modules/$kernelver/build/scripts/sign-file"
```

Comment:
Just a general note: my gut feeling is that we should remove all distribution logic out of dkms, in the long run.